### DISCUSSION TITLE
Displaying relative path instead of absolute path when adding new file or renaming file

### DISCUSSION THREAD

**Discussion by VaanNG** [nvim-tree/nvim-tree.lua#2992] [Q&A]
Hello,

I'm having trouble when trying to adding a new file or renaming a file in nvim-tree when the current absolute path is too long. I've navigated to current root project that I desire and nvim-tree opens up at that directory -- great! However, when adding a new file or renaming file, it still shows absolute path back to the User folder so I can't even see what I'm editing. Can someone help me on this?  😃 

![image](https://github.com/user-attachments/assets/2b39757d-21c6-4a34-9c66-2998987d303a)

**Comment by VaanNG** [comment]
Quick update: it would seem this only happens when I'm adding new file. Not renaming! 
![image](https://github.com/user-attachments/assets/da12392e-b65d-4c6a-b255-cf58e6979b4c)

**Comment by alex-courtis** [comment]
This looks very similar to #2989 which uses dressing for input dialogue. Please confirm that you're using dressing.

If that is the case, @devspectral and @VaanNG please raise an issue with dressing: https://github.com/stevearc/dressing.nvim/issues as it appears to be a recent bug.

**Reply by VaanNG** [reply]
Thank you for answer! I'd would seem the issue is currently with noice.nvim because that is what I'm using for that cmdline_popup!

**Reply by alex-courtis** [reply]
Thank you. I'd be most grateful if you raised it with them.
